packages/@react-aria/slider/src/useSliderThumb.ts

```diff
@@ -78,13 +78,12 @@ export function useSliderThumb(
   let reverseX = direction === 'rtl';
   let currentPosition = useRef<number>(null);
 
-  let pageSize = 1;
   let {keyboardProps} = useKeyboard({
     onKeyDown(e) {
       let {
         getThumbMaxValue,
         getThumbMinValue,
-        step
+        pageSize
```

Comment:
should `pageSize` default to `Math.max((getThumbMaxValue(index) - getThumbMinValue(index)) / 10), step)`?